Dockerfile.rhel7

```diff
@@ -2,16 +2,17 @@ FROM rhel7
 
 MAINTAINER Troy Dawson <tdawson@redhat.com>
 
-LABEL Name="openshift3/installer"
-LABEL Vendor="Red Hat" License=GPLv2+
-LABEL Version="v3.1.1.901"
-LABEL Release="6"
-LABEL BZComponent="aos3-installation-docker"
-LABEL Architecture="x86_64"
+LABEL name="openshift3/installer"
+LABEL vendor="Red Hat" License=GPLv2+
+LABEL version="v3.4.1.8"
+LABEL release="1"
+LABEL com.redhat.component="aos3-installation-docker"
+LABEL architecture="x86_64"
```

Comment:
Right - some changes will be necessary to this Dockerfile but there are other steps to do first, most notably to have a base playbook2image in the build system.
I'll revert that commit and leave Dockerfile.rhel7 untouched (just a rename of the previous Dockerfile)